cd/update_app_version.sh

```diff
@@ -58,7 +58,7 @@ new_version=$1
 app_version="$(getAppVersion)"
 
 if [ -z "${new_version}" ]; then
-    new_version="$(createNewVersion ${app_version})"
+    new_version="$(createNewVersion ${app_version}) # managed version"
```

Comment:
The new version shouldn't really have this attached to it. The managed version check should be done in the find and replace.